Shared/Samples/Show line of sight between geoelements/ShowLineOfSightBetweenGeoelementsView.swift

```diff
@@ -0,0 +1,358 @@
+    
+    var body: some View {
+        SceneView(
+            scene: model.scene,
+            graphicsOverlays: [model.graphicsOverlay],
+            analysisOverlays: [model.analysisOverlay]
+        )
+        .onDisappear {
+            model.stopAnimating()
+        }
+        .overlay(alignment: .top) {
+            HStack {
+                Text("Visibility:")
+                Text(model.visibilityStatus)
+            }
+            .frame(maxWidth: .infinity)
+            .padding(.vertical, 6)
+            .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+        }
+        .task {
+            await model.addGraphics()
+        }
+        .toolbar {
+            ToolbarItemGroup(placement: .bottomBar) {
+                Button("Settings") {
+                    isPresented = true
+                }
+                .sheet(isPresented: $isPresented) {
+                    settingsSheet
+                }
+            }
+        }
+        .errorAlert(presentingError: $model.error)
+    }
+    
+    /// The settings configuration sheet for adjusting the observer's height.
+    private var settingsSheet: some View {
+        NavigationStack {
+            Form {
+                let heightRange = 20.0...70.0
+                var numberFormat: FloatingPointFormatStyle<Double> {
+                    .init().precision(.fractionLength(0))
+                }
+                LabeledContent(
+                    "Observer Height",
+                    value: model.height,
+                    format: numberFormat
+                )
+                Slider(
+                    value: $model.height,
+                    in: heightRange,
+                    step: 1
+                ) {
+                    Text("Observer Height")
+                } minimumValueLabel: {
+                    Text(heightRange.lowerBound, format: numberFormat)
+                } maximumValueLabel: {
+                    Text(heightRange.upperBound, format: numberFormat)
+                }
+                .listRowSeparator(.hidden, edges: .top)
+            }
+            .presentationDetents([.fraction(0.25)])
+            .navigationTitle("Settings")
+            .navigationBarTitleDisplayMode(.inline)
+            .toolbar {
+                ToolbarItem(placement: .confirmationAction) {
+                    Button("Done") {
+                        isPresented = false
+                    }
+                }
+            }
+        }
+    }
+}
+
+private extension ShowLineOfSightBetweenGeoelementsView {
+    /// View model responsible for setting up the 3D scene, graphics, animation,
+    /// and performing line of sight analysis between moving and static geoelements.
+    @MainActor
+    @Observable
+    final class Model {
+        /// Set of predefined waypoints for animating the taxi's movement.
+        private let points = [
+            Point(x: -73.984513, y: 40.748469, spatialReference: .wgs84),
```

Comment:
I prefer [`init(latitude:longitude:)`](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/point/init(latitude:longitude:)) over [`init(x:y:z:m:spatialReference:)`](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/point/init(x:y:z:m:spatialreference:)) for clarity for 2D wgs84 points.